From 07e64d84ef508b1779d812c34955f443a84ba892 Mon Sep 17 00:00:00 2001
From: tommy-huang <tommy_huang@aspeedtech.com>
Date: Thu, 30 Sep 2021 15:09:12 +0800
Subject: [PATCH] drivers: i2c ipmb: fix warning and typo

Change some information from err into dbg.
Fix the data count error.

Signed-off-by: tommy-huang <tommy_huang@aspeedtech.com>
Change-Id: I9d7fd271b66f7bb372b07d4a9cd9d829a51793df
---
 drivers/i2c/slave/ipmb_slave.c   | 2 +-
 include/drivers/i2c/slave/ipmb.h | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/i2c/slave/ipmb_slave.c b/drivers/i2c/slave/ipmb_slave.c
index eca5bfdcd2..2634d9e592 100644
--- a/drivers/i2c/slave/ipmb_slave.c
+++ b/drivers/i2c/slave/ipmb_slave.c
@@ -136,7 +136,7 @@ int ipmb_slave_read(const struct device *dev, struct ipmb_msg **ipmb_data, uint8
 
 		return 0;
 	} else {
-		LOG_ERR("ipmb slave read: buffer empty!");
+		LOG_DBG("ipmb slave read: buffer empty!");
 		return 1;
 	}
 
diff --git a/include/drivers/i2c/slave/ipmb.h b/include/drivers/i2c/slave/ipmb.h
index c8701f4066..120f39b559 100644
--- a/include/drivers/i2c/slave/ipmb.h
+++ b/include/drivers/i2c/slave/ipmb.h
@@ -12,7 +12,7 @@
 
 #define GET_ADDR(addr)	((addr << 1) & 0xff)
 
-#define IPMB_MSG_DATA_LEN (IPMB_TOTAL_LEN - IPMB_REQUEST_LEN - 1)
+#define IPMB_MSG_DATA_LEN (IPMB_TOTAL_LEN - 1)
 
 struct ipmb_msg {
 	uint8_t rsSA;
-- 
2.17.1

